### PR TITLE
[ERSSUP-61175]-[]-[Added ContentType header in response and minor fix…

### PR DESCRIPTION
**Pull request by nhsd-angel-pastor**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/components/schemas/queryParameters/RevInclude.yaml [hunk 1]
```diff
@@ -11,7 +11,6 @@ description: |
   &category=http://snomed.info/sct|3457005
   &_revinclude=Appointment:based-on
   &_revinclude=Task:focus
-required: true
```
Comment (nhsd-jack-wainwright) [inline specification/components/schemas/queryParameters/RevInclude.yaml:14]: How does the endpoint behave if these `_revincludes` aren't included? Is the related appointment/task included always whether or not these are included?

Comment (nhsd-angel-pastor) [reply specification/components/schemas/queryParameters/RevInclude.yaml:14]: they're always included

File: specification/components/schemas/codings/ServiceRequestCategory.yaml [hunk 2]
```diff
@@ -15,7 +14,4 @@ items:
       example: 'http://snomed.info/sct'
     code:
       type: string
-      example: '3457005'
-    display:
-      type: string
-      example: 'Patient referral'
```
[the remark this answers is not in the document]
Comment (nhsd-angel-pastor) [reply specification/components/schemas/codings/ServiceRequestCategory.yaml:21]: we don't need to include it for this case

[the remark this answers is not in the document]
Comment (nhsd-angel-pastor) [reply specification/components/schemas/codings/ServiceRequestCategory.yaml:21]: we don't need to include it for this case

**Review by adamoldfield-nhs — APPROVED** [review]

**Pull request merged**